Tasks/DownloadPackageV1/multifilepackage.ts

```diff
@@ -49,7 +49,7 @@ export class MultiFilePackage extends Package {
     private async getPackageFileContent(fileMetadata: any): Promise<Map<string, PackageFileResult>> {
         return new Promise<Map<string, PackageFileResult>>(resolve => {
             var resultMap = new Map<string, PackageFileResult>();
-            resultMap[fileMetadata.name] = new PackageFileResult(fileMetadata.protocolMetadata.data.Content, false);
```

Comment:
https://github.com/microsoft/azure-pipelines-tasks/pull/16154 are these file changes in a different PR for master?

Reply:
Here is that change: https://github.com/microsoft/azure-pipelines-tasks/pull/16146